ref_test.go

```diff
@@ -0,0 +1,24 @@
+package hype
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Ref_MarshalJSON(t *testing.T) {
+	t.Parallel()
+
+	r := require.New(t)
```

Comment:
I'm curious if we need this any more. I think the `assert` and `require` libraries just allow us to do this.  

Like `require.NoError(t, err)` etc.  Thoughts?